src/CommandHandler.cpp

```diff
@@ -465,6 +487,7 @@ void CommandHandler::handle(std::string const &Command) {
   LOG(Sev::Warning, "Could not understand this command: {}", Command);
 }
 
+/// Helper to get nicer error messages.
```

Comment:
could be "helper to get nested error messages showing traceback"

Reply:
The function name pretty much says that ;)